spec/rubocop/node_pattern_spec.rb

```diff
@@ -86,6 +86,14 @@
     end
   end
 
+  describe 'literals' do
+    context 'negative integer literals' do
+      let(:pattern) { '(int -100)' }
+      let(:ruby) { '-100' }
+      it_behaves_like :matching
+    end
+  end
+
```

Comment:
Is it possible to also add negative float literals? `(float 1.0)` is supported at the moment, but not `(float -1.0)`.